src/IteratorToArrayConvertCapable.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace RebelCode\Bookings\WordPress\Module;
+
+use Dhii\Util\Normalization\NormalizeIterableCapableTrait;
+use stdClass;
+use Traversable;
+
+/**
+ * Trait IteratorToArrayConvertCapable, adds ability to convert an iterator to an array.
+ *
+ * @since [*next-version*]
+ */
+trait IteratorToArrayConvertCapable
+{
+    /* @since [*next-version*] */
+    use NormalizeIterableCapableTrait;
```

Comment:
This is not so ideal. The best approach for these kinds of traits is to add the method that you want to use in this trait as abstract. In this case, an abstract `_normalizeIterable()`. You can simply copy-paste the method from the `NormalizeIterableCapableTrait` and remove its body and mark it as abstract. Then, where it is used (such as in the `SettingsContainer`), also `use NormalizeIterableCapableTrait`, if not already used.